lib/bibliothecary/parsers/go.rb

```diff
@@ -95,6 +105,39 @@ def self.parse_dep_lockfile(file_contents)
         map_dependencies(manifest, 'projects', 'name', 'revision', 'runtime')
       end
 
+      def self.parse_go_mod(file_contents)
+        deps = []
+        file_contents.lines.map(&:strip).each do |line|
+          next if line.start_with?("module ")
+          next if line.start_with?("go ")
+          next if line.start_with?("require (")
+          next if line.start_with?(")")
+          match = line.gsub(/(\/\/(.*))/, '').match(GOMOD_REGEX)
```

Comment:
Is this for the comment stuff at the end?

Reply:
Yeah, there's an `// indirect` that gets added sometimes, and I'm not sure yet if there are other comments besides `indirect` so I just did a catch-all. Here's a note about it ([ref](https://github.com/golang/go/wiki/Modules#faqs--gomod-and-gosum)):

```If a dependency of your module does not itself have a go.mod (e.g., because the dependency has not yet opted in to modules itself), or if its go.mod file is missing one or more of its dependencies (e.g., because the module author did not run go mod tidy), then the missing transitive dependencies will be added to your module's requirements, along with an // indirect comment to indicate that the dependency is not from a direct import within your module.```